src/web/tools/PointerEventManager.ts

```diff
@@ -200,7 +217,16 @@ export default class PointerEventManager extends EventManager<HTMLElement> {
     );
   }
 
-  protected mapEvent(event: PointerEvent, eventType: EventTypes): AdaptedEvent {
+  protected mapEvent(
+    event: PointerEvent | WheelEvent,
+    eventType: EventTypes
+  ): AdaptedEvent {
+    return event instanceof PointerEvent
+      ? this.mapPointerEvent(event, eventType)
+      : this.mapWheelEvent(event);
```

Comment:
We may want to redesign the event manager system in the future. We now have `PointerEventManager` that needs to check whether the event it receives is a `PointerEvent` which seems wrong to me.

Reply:
There's nothing wrong with adding another event manager, but I'm not exactly sure how would it work. Adding `MouseEventManager` doesn't make much sense at this point for me. Also, this manager would have only one method - `onWheel`.

On the other hand, `KeyboradManager` also has one callback, so maybe it is fine 😅

Reply:
Okay, I've discussed it with @latekvo and we decided that it will be ok to add `WheelEventManager`. Done in [bf008b1](https://github.com/software-mansion/react-native-gesture-handler/pull/3163/commits/bf008b19202cc5d23b4bfa6c68fd4eec32d8c5ce)